lib/spack/spack/test/concretize_requirements.py

```diff
@@ -856,6 +856,8 @@ def test_skip_requirement_when_default_requirement_condition_cannot_be_met(
 
 def test_requires_directive(concretize_scope, mock_packages):
     compilers_yaml = pathlib.Path(concretize_scope) / "compilers.yaml"
+
+    # NOTE: target is omitted here so that the test works on aarch64, as well.
```

Comment:
This can probably be extracted as a smaller fix

Reply:
done in #41943